detections/endpoint/detect_critical_alerts_from_security_tools.yml

```diff
@@ -0,0 +1,47 @@
+name: Detect Critical Alerts from Security Tools
+id: 483e8a68-f2f7-45be-8fc9-bf725f0e22fd
+version: 1
+date: '2024-06-04'
+author: Gowthamaraj Rajendran, Patrick Bareiss, Splunk
+status: production
+type: TTP
+data_source: ["windows defender logs"]
```

Comment:
lets add a data source file for this data in the data sources folder and use that name in here